.bazelci/presubmit.yml

```diff
@@ -0,0 +1,12 @@
+---
+# TODO(yannic): Ideally, we should also enable buildifier and all platforms should test `//...`.
```

Comment:
please add more info about what is the purpose of this file and who to reach out to with questions.
Can you also add a link to the test results? (Once known)

Reply:
Thanks, done.

I'll send a follow-up with the link when it's known (Filed https://github.com/bazelbuild/continuous-integration/issues/857 to create the pipeline)